md-docs/FAQ.md

```diff
@@ -9,20 +9,20 @@ repository.
 The GitHub teams relevant to this repository are as follows (Note, you
 won't necessarily have access to see these links):
 
-- [GetOpenJDK](https://github.com/orgs/AdoptOpenJDK/teams/getopenjdk) - `Triage` level of access which lets you assign issues to people
-- [build](https://github.com/orgs/AdoptOpenJDK/teams/build) - `Write` access which lets you approve and merge PRs and run and configure most Jenkins jobs
-- [release](https://github.com/orgs/AdoptOpenJDK/teams/build) - Allows you to run the release jobs in Jenkins
+- GetOpenJDK- `Triage` level of access which lets you assign issues to people
```

Comment:
There are actually named teams for this repo.  Suggest looking at the AdoptOpenJDK/Teams tab in GitHub and see if you can transpose that info here.

Reply:
I can't see the teams tab. 
![image](https://user-images.githubusercontent.com/70375257/115200732-70cad880-a0ec-11eb-9ae9-541b8c399f13.png)

Reply:
I don't see the teams tab. 
![image](https://user-images.githubusercontent.com/70375257/115200936-abcd0c00-a0ec-11eb-89b7-4d5ec9364a79.png)

Reply:
could it be because I'm not part of any team?